contracts/LifCrowdsale.sol

```diff
@@ -30,6 +33,10 @@ contract LifCrowdsale is Ownable, Pausable {
 
   uint256 public tokensSold;
 
+  uint256 public totalPresaleWei;
```

Comment:
let's put a comment describing what this var means, and the same for the next var? This could be smth like "total amount of wei received as presale payments (both private and public)"